packages/common/dto/skill-information.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { CurrentEmploymentTypes, RegistrationStatus, StreamTypes } from '@ehpr/common/form-payload';
+import { IsIn, IsNumber, IsOptional, IsString, Length, Max, Min } from 'class-validator';
+
+export class SkillInformationDTO {
+  @IsString()
+  @IsIn(Object.values(StreamTypes))
+  streamTypes!: StreamTypes;
+
+  @IsString()
+  @IsIn(Object.values(RegistrationStatus))
+  registrationStatus!: RegistrationStatus;
+
+  @IsNumber()
+  @Max(99999999)
+  @Min(0)
+  registrationNumber!: number;
```

Comment:
Discussion point: I'd rather represent this as a string if we're not doing any math with it, the min/max is a smell to me I think `IsNumberString` can be used

Reply:
Is this the registration number that is autogenerated or something that the user fills in?

Reply:
That's totally fine. As stated above, these validators are just a starting point. I figure we'll refine them to be more in line with what makes sense for the front end to send.